src/reducers/layout.js

```diff
@@ -1,16 +1,33 @@
 import {
-  DEFAULT_AMPLITUDE_VALUE,
+  DEFAULT_AMPLITUDE,
```

Comment:
At some point maybe we should break out the non-layout part of this reducer into a name that makes more sense... `lab`?

Reply:
created an issue with this https://github.com/graasp/graasp-app-radiating-charge/issues/28